picoquic/tls_api.c

```diff
@@ -1288,3 +1288,7 @@ int picoquic_create_cnxid_reset_secret(picoquic_quic_t* quic, picoquic_connectio
 
     return (ret);
 }
+
+void picoquic_tls_enable_client_authentication(picoquic_quic_t* quic) {
+    ((ptls_context_t*)quic->tls_master_ctx)->client_authentication = 1;
+}
```

Comment:
Authentication mode parameter would fit naturally here.

Also, this line causes the Travis build to break:
~~~
/home/travis/build/private-octopus/picoquic/picoquic/tls_api.c: In function ‘picoquic_tls_enable_client_authentication’:
/home/travis/build/private-octopus/picoquic/picoquic/tls_api.c:1293:44: error: ‘ptls_context_t’ has no member named ‘client_authentication’
     ((ptls_context_t*)quic->tls_master_ctx)->client_authentication = 1;
~~~
Do you have a dependency on a PR in progress in Picotls?

Reply:
Yeah, I linked the pull request.